### DISCUSSION TITLE
hexo-next太久没有更新惹

### DISCUSSION THREAD

**Discussion by woruo27** [walinejs/waline#494] [Q&A] [answered]
所以hexo-next是随着realase进行瀑布式开发？

**Comment by lizheming** [comment]
hexo-next 一般没有特殊需求不怎么需要发布的，所以不会随着客户端版本发布

**Comment by Mister-Hope** [comment, marked as answer]
因为hexo-next是用cdn拉最新的客户端版本，所以没有破坏性更改是不需要变动的，目前v3版本已经发了。
